docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -93,6 +93,18 @@ both of the following services:
 + Master RPC on port 19998
 + Worker RPC on port 29999
 
+Within the Alluxio cluster, please also make sure the master and worker containers can reach
+each other on the following ports.
+You can find all the ports used by Alluxio components in [Alluxio configurations]({{ '/en/reference/Properties-List.html?q=port' | relativize_url }})
+
++ Master RPC on port 19998
++ Master embedded journal on port 19200 if you are using [Internal leader selection]({{ '/en/deploy/Running-Alluxio-On-Docker.html#option-a-internal-leader-election' | relativize_url }})
++ Job master RPC on port 20001
++ Job master embedded journal on port 20003 if you are using [Internal leader selection]({{ '/en/deploy/Running-Alluxio-On-Docker.html#option-a-internal-leader-election' | relativize_url }})
++ Worker RPC on port 29999
++ Job worker RPC on port 30001
++ Job worker data on port 30001
+
```

Comment:
We can also refer users to the `Requirements` page that lists out all of these ports? I think it would be better to do that rather than having the information duplicated

Reply:
Good call. The table of contents in missing on the `Requirements` page. This will be fixed once https://github.com/Alluxio/alluxio/pull/11438 is in.

Reply:
I still think we should remove this section?

Reply:
Yes PTAL at the latest change. #11438 fixes the Requirements.html page by adding the table of contents (and generates the missing links).